src/errors/test-run/templates.js

```diff
@@ -92,6 +92,18 @@ export default {
          ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.actionListArgumentError]: err => markup(err, `
+        The <code>${err.argumentName}</code> argument is expected to be an non-empty array, but it was <code>${err.actualValue}</code>.
+
+        ${err.getCallsiteMarkup()}
+    `),
+
+    [TYPE.actionListArgumentLengthError]: err => markup(err, `
```

Comment:
Why we need two errors for this? Can we just have `actionNonEmptyArrayArgumentError` which will cover both cases?